### ISSUE TITLE
NamespaceResolver does not correctly lookup namespace for properties in parent class

### ISSUE THREAD

**Issue by template-provider** [JsonMapper/JsonMapper#116]
**Describe the bug**
I want to map a class which extends another class and has property with phpdoc (class).

```php
<?php
declare(strict_types = 1);

namespace Foo

use Foo\Model\BarModel

class test extends BarModel
{
}
```

```php
<?php
declare(strict_types = 1);

namespace Foo\Model

use Foo\MetaModel\Meta;

class BarModel
{
    /** @var null|Meta */
    public $meta;
}
```

```php
<?php

declare(strict_types = 1);

namespace Foo\MetaModel;

class Meta
{
    /** @var null|string */
    public $type;
}

```
The Namespaceresolver does not correct lookup the namespace of the used Foo\MetaModel\Meta.
Instead it will look only for Meta. Only a complete phpdoc helps here with Foo\MetaModel\Meta

PHP Version: 7.2
Json-Mapper: 2.9.0
Mode: (new JsonMapperFactory())->bestFit()

**Comment by DannyvdSluijs** [comment]
Thanks for the extensive report. This really helps in finding the root cause. 

Is this behavior that used to work in previous version that you know?

**Comment by template-provider** [comment]
I think it was broken before too

**Comment by DannyvdSluijs** [comment]
I was able to reproduce the problem and I already have a hunch why this occurs.
The `NamespaceResolver` is only looking at the class file and not at the files of the classes it is extending.

**Comment by DannyvdSluijs** [comment]
I've just released v2.9.1 which solves the issue of imports in parent classes.

**Comment by template-provider** [comment]
thx ... seems to work
